client/src/Components/CommonComponent/Header/index.css

```diff
@@ -0,0 +1,35 @@
+.App-header {
+  height: 3.6875 rem;
```

Comment:
remove space between number and rem unit
like this
3.6875rem 

Try to solve this in all other rem